.github/workflows/site.yaml

```diff
@@ -24,8 +24,8 @@ jobs:
           fetch-depth: 0
       - uses: astral-sh/setup-uv@v3
       - run: uv python install
-      - run: make build install
       - uses: quarto-dev/quarto-actions/setup@v2
+      - run: make build install
```

Comment:
Is this change intensional?

Reply:
Yeah, trying to figure out what the GH Action is broken...

Reply:
See https://github.com/posit-dev/posit-sdk-py/pull/407

Reply:
🙏 thanks!